Pipfile

```diff
@@ -17,6 +16,7 @@ psycopg2 = "*"
 logstash-formatter = "*"
 python-dateutil = "*"
 pytz = "*"
+kerlescan = {editable = true, ref = "0.55", git = "https://github.com/RedHatInsights/kerlescan.git"}
```

Comment:
any reason for the following changes?
- the line order from line 6 to 19? (it was alphabetical order before)
- the order of these fields (editable, ref, git) is also changed, previously it was alphabetical order (editable, git, ref) previously

Reply:
I used the `pipenv uninstall kerlescan` command, followed by `pipenv install --editable git+https://github.com/RedHatInsights/kerlescan.git@0.55#egg=kerlescan` command to get pipenv to install the correct version of kerlescan into my env.  Without doing this, regardless of running `pipenv update`, `pipenv lock`, `pipenv install --dev`, etc, running the unit tests would complain about not finding the kerlescan module.
The `pipenv install ...` command put the args in this order and the entry in the position to where it was moved.

I have updated the order so kerlescan is back in alphbetical order with the other modules.  In system-baseline-backend the modules were not in alphabetical order to begin with.

Reply:
Thanks